src/icons/index.ts

```diff
@@ -22,6 +26,7 @@ export { ReactComponent as Plus } from './images/PlusIcon.svg';
 export { ReactComponent as Refresh } from './images/RefreshIcon.svg';
 export { ReactComponent as Search } from './images/SearchIcon.svg';
 export { ReactComponent as Settings } from './images/SettingsIcon.svg';
+export { ReactComponent as SmallLightLogo } from './images/small-light-logo.svg';
```

Comment:
No need for duplicating this icon - right?   It should be available as a component under logo/branding?   I'd check with Ed on what's there for you to use for a small light logo.

Reply:
This icon reference has been revoked